global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "8.0.200",
```

Comment:
Is updating the SDK necessary? (feels weird to move to a STS instead of LTS if there isn't a pressing need for the new SDK features)

Reply:
Switched back to LTS.